src/main/resources/db.changeLog/changeLog-master.xml

```diff
@@ -2,7 +2,40 @@
 <databaseChangeLog
         xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xmlns:pro="http://www.liquibase.org/xml/ns/pro"
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
-      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
-    <includeAll path="com/example/changelog/"/>
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd
+		http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+		http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-latest.xsd">
+    <preConditions>
+        <runningAs  username="liquibase"/>
+    </preConditions>
+
+    <changeSet id ="1" author="alex">
+        <createTable tableName="category">
+            <column name = "id" type="long" autoIncrement="true">
+                <constraints primaryKey="true" />
+            </column>
+            <column name = "name" type="varchar(255)">
+                <constraints unique="true"/>
+            </column>
+            <column name = "description" type="varchar(1000)"/>
+        </createTable>
+        <createTable tableName="item">
+            <column name = "id" type="long" autoIncrement="true">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name = "name" type="varchar(255)"/>
+            <column name = "category_id" type="long">
+                <constraints foreignKeyName="category_id" references="category(id)"/>
+            </column>
+            <column name="description" type="varchar(2000)"/>
+            <column name="image_src" type="varchar(500)"/>
+        </createTable>
+    </changeSet>
+
+
+
+
 </databaseChangeLog>
```

Comment:
is it good practice to leave one empty row (new line) in the end of the each file